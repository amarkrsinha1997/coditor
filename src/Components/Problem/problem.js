import React from "react";
import PropTypes from "prop-types";

const Problem = props => {
  const content = props.content;
  return (
    <div className="content" dangerouslySetInnerHTML={{ __html: content }} />
  );
};

Problem.propTypes = {
  content: PropTypes.string.isRequired
};
export default Problem;
