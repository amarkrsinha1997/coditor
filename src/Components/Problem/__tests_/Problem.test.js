import React from "react";
import renderer from "react-test-renderer";
import Problem from "./../Problem";

describe("<TestResults /> ", () => {
  it("should render properly", () => {
    const tree = renderer.create(
      <Problem
        content={'<div>Problem statement</div>'}
      />
    );
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
