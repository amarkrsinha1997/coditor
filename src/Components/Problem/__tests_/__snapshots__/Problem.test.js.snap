// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TestResults />  should render properly 1`] = `
<div
  className="content"
  dangerouslySetInnerHTML={
    Object {
      "__html": "<div>Problem statement</div>",
    }
  }
/>
`;
